tests/ops/functions/test_equal.py

```diff
@@ -107,6 +108,66 @@
 )
 
 
+PARAMETRIZED_MEASUREMENTS = [
+    qml.sample(qml.PauliY(0)),
+    qml.sample(wires=0),
+    qml.sample(),
+    qml.counts(qml.PauliZ(0)),
+    qml.counts(wires=[0, 1]),
+    qml.counts(wires=[1, 0]),
```

Comment:
I added these two tests with swapped wires since the order of the wires matters (according to [this](https://github.com/PennyLaneAI/pennylane/blob/master/pennylane/wires.py#L134)), so currently these two measurements are not equal. Is this ok? or should we add logic to sort wires and have  them be equal?

Reply:
This is definitely ok! Using different wire order will give different results